keel-intents/src/main/kotlin/com/netflix/spinnaker/keel/intents/processors/SecurityGroupIntentProcessor.kt

```diff
@@ -73,21 +73,20 @@ class SecurityGroupIntentProcessor
           trigger = Trigger(intent.id)
         )
       ),
+      // TODO rz - It'd be rad to enumerate what actually changed in the reason if it's a delta, rather than giving
+      // a generic message.
       ConvergeReason.CHANGED.reason
     )
   }
 
   private fun getSecurityGroups(spec: SecurityGroupSpec): Set<SecurityGroup> {
-    // TODO rz - Quite dislike that any new cloudprovider would need to edit keel for declarative support... how to fix?
-    // Maybe this is the sacrifice we need to make to start statically typing all of our models?
     if (spec is AmazonSecurityGroupSpec) {
```

Comment:
Any thoughts about how we're gonna handle this for other types of security groups?

Reply:
Each cloud provider will need to bring their own implementation at this point, just as they need to bring their own implementation into orca stages.